etc/institutions-auth.xsl

```diff
@@ -109,13 +109,25 @@
                         <user>
                             <!--  Each institution has its customized mapping of attributes  -->
                             <username><xsl:value-of select="//attribute[@name='mail']/@value"/></username>
-                            <fullname><xsl:value-of select="//attribute[@name='displayedName']/@value"/></fullname>
+                            <fullname><xsl:value-of select="//attribute[@name='displayName']/@value"/></fullname>
                             <familyName><xsl:value-of select="//attribute[@name='sn']/@value"/></familyName>
                             <givenName><xsl:value-of select="//attribute[@name='givenName']/@value"/></givenName>
                             <middleNames/>
                             <suffix/>
                         </user>
                     </xsl:when>
+                    <!-- Concordia College (CORD) -->
+                    <xsl:when test="$idp='cord'">
+                        <id>cord</id>
+                        <user>
+                            <username><xsl:value-of select="//attribute[@name='mail']/@value"/></username>
+                            <fullname><xsl:value-of select="//attribute[@name='displayName']/@value"/></fullname>
+                            <familyName />
+                            <givenName />
+                            <middleNames/>
+                            <suffix/>
+                        </user>
+                    </xsl:when>
```

Comment:
* Step 5: Update `institutions-auth.xsl` so that CAS can normalize raw CAS profile attributes (or SAML attributes if using SAML / Shibboleth in the other case) before sending it to OSF via API.
  * Step 5.1: Update the private server settings to include this one.